bin/update-repos

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+git checkout master
+git pull origin master
+
+cd ../pacta-data
+git checkout master
+git pull origin master
+
+cd ../create_interactive_report
+git checkout master
+git pull origin master
+
+cd ../StressTestingModelDev
+git checkout master
+git pull origin master
+
+cd ../PACTA_analysis
+git checkout "$current_branch"
```

Comment:
not really sure what all this is for, but I hope this is not intended to run on the web server every time a report generation is triggered because I imagine this will be a large bottleneck

Reply:
I think I understand what this is for now, and in principle, I get it. But it seems a bit dangerous to me to jump into 4 other directories/repos and perform git commands automatically. For example... what if one of these local repos is on a different branch and has uncommitted changes?

Reply:
Yeah, it's probabbly going to cause more problems than solutions. And a better approach I think would be to create a temporary directory, and clone the repo -- but that would take to long for the quick iteration I like for testing.

I'll just remove this script.